subprojects/core/src/integTest/groovy/org/gradle/api/ConfigurationOnDemandIntegrationTest.groovy

```diff
@@ -76,6 +83,7 @@ class ConfigurationOnDemandIntegrationTest extends AbstractIntegrationSpec {
         fixture.assertProjectsConfigured(":")
     }
 
+    @ToBeFixedForIsolatedProjects(because = "Allprojects")
```

Comment:
💅 Let's preserve the incompatible API spelling as _allprojects_ for easier searching, just in case